### PR TITLE
Fixing the server load call and amqp cast_with_consumer

### PR DESCRIPTION
**Pull request by hub-cap**
- Fixing a small bug in cast_with_consumer (copy pasta!)
- Small fixes to the load call to make it work

### CODE REVIEW

**Comment by cp16net** [comment]
+1

**Comment by TimSimpsonR** [comment]
+1

File: reddwarf/db/sqlalchemy/mappers.py [hunk 1]
```diff
@@ -30,6 +30,8 @@ def map(engine, models):
     orm.mapper(models['instance'], Table('instances', meta, autoload=True))
     orm.mapper(models['service_image'],
                Table('service_images', meta, autoload=True))
+    orm.mapper(models['service_statuses'],
+               Table('service_statuses', meta, autoload=True))
```
Comment (TimSimpsonR) [inline reddwarf/db/sqlalchemy/mappers.py:34]: At last the mystery is solved.

Comment (cp16net) [inline reddwarf/db/sqlalchemy/mappers.py:34]: nice!

**Comment by ed-** [comment]
+1

**Comment by rnirmal** [comment]
+1

**Pull request merged**